src/components/Card.js

```diff
@@ -1,59 +1,84 @@
 export default class Card {
-  constructor({ title, link }, cardSelector, handleImageClick) {
-    this._title = title;
+  constructor({
+    name,
+    link,
```

Comment:
Needs correction: I can't delete cards on the server. When I click delete button the confirm popup should be opened

Reply:
The confirm popup now opens but still working to fix the delete functionality.